lib/fog/compute/google.rb

```diff
@@ -18,153 +18,153 @@ class Google < Fog::Service
 
-      request :set_metadata
-      request :set_tags
-      request :set_forwarding_rule_target
-      request :set_global_forwarding_rule_target
-      request :set_target_http_proxy_url_map
-
-      request :add_target_pool_instances
-      request :add_target_pool_health_checks
-      request :add_backend_service_backends
-      request :add_url_map_host_rules
-      request :add_url_map_path_matchers
-      request :add_zone_view_resources
-      request :add_region_view_resources
-      request :add_instance_group_instance
+      request :list_addresses
+      request :list_aggregated_addresses
+      request :list_aggregated_disk_types
+      request :list_aggregated_disks
+      request :list_aggregated_instance_groups
+      request :list_aggregated_machine_types
+      request :list_aggregated_servers
+      request :list_aggregated_subnetworks
+      request :list_aggregated_target_instances
+      request :list_backend_services
+      request :list_disk_types
+      request :list_disks
+      request :list_firewalls
+      request :list_forwarding_rules
+      request :list_global_forwarding_rules
+      request :list_global_operations
+      request :list_http_health_checks
+      request :list_images
+      request :list_instance_group_instances
+      request :list_instance_groups
+      request :list_machine_types
+      request :list_networks
+      request :list_region_operations
+      request :list_region_view_resources
+      request :list_region_views
+      request :list_regions
+      request :list_routes
+      request :list_servers
+      request :list_snapshots
+      request :list_subnetworks
+      request :list_target_http_proxies
+      request :list_target_instances
+      request :list_target_pools
+      request :list_url_maps
+      request :list_zone_operations
+      request :list_zone_view_resources
+      request :list_zone_views
+      request :list_zones
 
-      request :remove_target_pool_instances
+      request :remove_instance_group_instances
       request :remove_target_pool_health_checks
+      request :remove_target_pool_instances
+
       request :set_common_instance_metadata
-      request :remove_instance_group_instance
+      request :set_forwarding_rule_target
+      request :set_global_forwarding_rule_target
+      request :set_metadata
+      request :set_server_disk_auto_delete
+      request :set_server_scheduling
+      request :set_tags
+      request :set_target_http_proxy_url_map
 
       request :attach_disk
       request :detach_disk
-      request :get_server_serial_port_output
       request :reset_server
-      request :set_server_disk_auto_delete
-      request :set_server_scheduling
-      request :add_server_access_config
-      request :delete_server_access_config
-      request :update_url_map
-      request :validate_url_map
       request :start_server
       request :stop_server
+      request :update_url_map
+      request :validate_url_map
```

Comment:
Well, it was in a bit of a mess, so I decided to clean the list up a bit, why not :-P